builtin/providers/github/resource_github_membership_test.go

```diff
@@ -0,0 +1,84 @@
+package github
+
+import (
+	"fmt"
+	"github.com/google/go-github/github"
+	"github.com/hashicorp/terraform/helper/resource"
+	"github.com/hashicorp/terraform/terraform"
+	"testing"
+)
+
+func TestAccGithubMembership_basic(t *testing.T) {
+	var membership github.Membership
+
+	resource.Test(t, resource.TestCase{
+		PreCheck:     func() { testAccPreCheck(t) },
+		Providers:    testAccProviders,
+		CheckDestroy: testAccCheckGithubMembershipDestroy,
+		Steps: []resource.TestStep{
+			resource.TestStep{
+				Config: testAccGithubMembershipConfig,
+				Check: resource.ComposeTestCheckFunc(
+					testAccCheckGithubMembershipExists("github_membership.test_org_membership", &membership),
+				),
```

Comment:
I would add another function here to this `ComposeTestCheckFunc` that verifies the value of `membership.Role` is what you expected it to be.